.github/workflows/release_cli.yml

```
@@ -226,13 +244,29 @@ jobs:
             ./packages/@biomejs/wasm-web
           if-no-files-found: error
 
+  smoke-test-win32-arm64:
+    name: Smoke test win32-arm64
+    # depot doesn't offer depot arm runners
+    runs-on: windows-11-arm
+    needs: build
+    if: needs.check.outputs.version_changed == 'true' || needs.check.outputs.nightly == 'true'
```

actionlint (1.7.12)

[error] property "check" is not defined in object type {build: {outputs: {prerelease: string; version: string}; result: string}}

(expression)